From e04338d7e7987bd3d4c63579dd76e3ccbaf8c534 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Fri, 24 Aug 2018 14:36:15 +0800
Subject: [PATCH 4679/5242] MLK-19329 ARM64: dts: imx8mm-evk: disable usbotg2
 data

commit  ff8a11ea5581e5fd2acf5f23a5499d6ebdbff086 from
https://source.codeaurora.org/external/imx/linux-imx.git

As the typec2(usbotg2) is always with charger on, which makes the usbotg2
can't enter runtime suspend, thus high bus can't be released, disable it
for now and will improve high bus only for data communication.

Signed-off-by: Li Jun <jun.li@nxp.com>
(cherry picked from commit 5b4290c370064cf9aa3a5e5c0cd12194f42aea7a)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index 365e2d6..8cc9240b 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -947,7 +947,7 @@
 	extcon = <0>, <&typec2_ptn5110>;
 	picophy,pre-emp-curr-control = <3>;
 	picophy,dc-vol-level-adjust = <7>;
-	status = "okay";
+	status = "disabled";
 };
 
 &usdhc1 {
-- 
1.7.9.5

